src/main/java/org/sonatype/gradle/plugins/scan/ossindex/DependencyGraphResponseHandler.java

```diff
@@ -50,12 +51,16 @@ public boolean handleOssIndexResponse(
       Set<ResolvedDependency> dependencies,
       Map<ResolvedDependency, PackageUrl> dependenciesMap,
       Map<PackageUrl, ComponentReport> response) {
-
-    log.info(BannerUtils.createBanner());
-    log.info("Checking vulnerabilities in {} dependencies", dependenciesMap.size());
-
     boolean hasVulnerabilities = false;
 
+    if (!extension.isShowAll()) {
+      dependenciesMap = getDependenciesMapWithVulnerabilities(dependenciesMap, response);
+      if (dependenciesMap.isEmpty()) {
+        log.info("No vulnerabilities found!");
+        return false;
+      }
```

Comment:
Suggestion. Might be good to add logging how many vulnerabilities where found. Just to be consistent with the default / table report.
```else {
        log.info("Found vulnerabilities in {} dependencies", dependenciesCount);
       return true;
      }
```

Reply:
> Suggestion. Might be good to add logging how many vulnerabilities where found (see below). Just to be consistent with the default / table report.
> 
> ```
> else {
>         log.info("Found vulnerabilities in {} dependencies", dependenciesCount);
>        return true;
>       }
> ```

Done: 8301d50af33e54a71e450a6287b9fd92d756c475